CMP1903M A01 2223/Testing.cs

```diff
@@ -10,8 +10,47 @@ public class Testing
     {
         public static void Test()
         {
+            //This Displays the pack
+            Console.WriteLine("This is card pack: ");
```

Comment:
Output could have been handled better to present the output nicely e.g having spaces before each operation